src/encoded/static/components/regulome_search.js

```diff
@@ -257,21 +258,42 @@ ExampleEntry.propTypes = {
     handleExample: PropTypes.func.isRequired,
 };
 
+const AssemblySelector = props => (
+    <div className="assembly-switch">
+        <div className={`switch-label ${props.selection === 'GRCh38' ? 'active' : ''}`}>GRCh38</div>
+        <label className="switch" id="assembly-switch">
+            <input
+                type="checkbox"
+                onChange={() => { props.onSelection(props.selection); }}
+                checked={props.selection === 'hg19'}
+            />
+            <span className="slider round" />
+        </label>
```

Comment:
While I haven’t run this code, I’m guessing this checkbox isn’t accessible. The `<label>` doesn’t have speakable text in it.